pkgs/tools/security/enpass/default.nix

```diff
@@ -0,0 +1,86 @@
+{stdenv, fetchurl, dpkg, patchelf, openssl, xorg
+, glib, mesa, libpulseaudio, zlib, dbus, fontconfig, freetype
+, gtk2, pango, atk, cairo, gdk_pixbuf, jasper, xkeyboardconfig
+, makeWrapper , makeDesktopItem, python}:
+let
+  data = import ./data.nix;
+  # used of both wrappers and libpath
+  uselibs = with xorg; [
+    openssl
+    mesa
+    fontconfig
+    freetype
+    libpulseaudio
+    zlib
+    dbus
+    libX11
+    libXi
+    libSM
+    libICE
+    libXext
+    libXrender
+    libXScrnSaver
+    glib
+    gtk2
+    pango
+    cairo
+    atk
+    gdk_pixbuf
+    jasper
+  ];
+    libPath = stdenv.lib.makeLibraryPath ([
+      stdenv.cc
+      stdenv.cc.cc
+    ] ++ uselibs);
+in stdenv.mkDerivation rec {
+
+    inherit (data) version;
+    name = "enpass-${version}";
+
+    desktopItem = makeDesktopItem {
+      name = "Enpass";
+      exec = "$out/bin/Enpass";
+      #icon = "Enpass";
+      comment = "password manager";
+      desktopName = "Enpass";
+      genericName = "Password manager";
+      categories = "Application;Development;";
+    };
+
+
+    src = fetchurl {
+      inherit (data) sha256;
+      url = "http://repo.sinew.in/${data.path}";
+    };
+
+    meta = {
+      description = "a well known password manager";
+      homepage = https://www.enpass.io/;
+      license = stdenv.lib.licenses.unfree;
+      platforms = stdenv.lib.platforms.linux;
+    };
+
+    buildInputs = [makeWrapper dpkg patchelf ];
+    phases = [ "unpackPhase" "installPhase" ];
```

Comment:
Are you sure this is considered bad practice when packaging binaries? I find it acceptable here (but I might be mistaken)